Detectors/TPC/reconstruction/src/CTFCoder.cxx

```diff
@@ -0,0 +1,88 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   CTFCoder.cxx
+/// \author ruben.shahoyan@cern.ch
+/// \brief class for entropy encoding/decoding of TPC compressed clusters data
```

Comment:
more descriptive naming in local variables.

Reply:
which variable?

Reply:
again very much my personal opinion. I had to look for couple of seconds to understand what was going on and felt that more verbose variable names would make the code more readable. 
`h` could be something like `counters` and `c` something like `compClusters` and `sz` something like `size`.  But I see how shorter names can help focus on what is important.